h/views/api/bulk/annotation.py

```diff
@@ -40,7 +40,9 @@ def bulk_annotation(request):
             # Use the authority from the authenticated client to ensure the user
             # is limited to items they have permission to request
             authority=request.identity.auth_client.authority,
-            **query_filter,
```

Comment:
I reckon kwargs are more confusing here than the alternative.

If we add/remove/modify parameters using them explicitly will have the tests fail in each line where we need to update due to the change.